src/RA_Condition.cpp

```diff
@@ -10,10 +10,7 @@ static_assert( SIZEOF_ARRAY( COMPARISONTYPE_STR ) == NumComparisonTypes, "Must m
 const char* CONDITIONTYPE_STR[] = { "", "Pause If", "Reset If", "Add Source", "Sub Source", "Add Hits"};
 static_assert( SIZEOF_ARRAY( CONDITIONTYPE_STR ) == Condition::NumConditionTypes, "Must match!" );
 
-int g_AddBuffer = 0;
-int g_AddHits = 0;
```

Comment:
these are now passed from `ConditionGroup` to `Condition`, instead of being global.